beacon-chain/sync/validate_beacon_blocks.go

```diff
@@ -204,7 +204,9 @@ func (s *Service) validateBeaconBlockPubSub(ctx context.Context, pid peer.ID, ms
 	// Log the arrival time of the accepted block
 	startTime, err := slots.ToTime(genesisTime, blk.Block().Slot())
 	if err != nil {
-		return pubsub.ValidationIgnore, err
+		// return without logging
+		err = nil
+		return pubsub.ValidationAccept, err
```

Comment:
Why don't we log the error and continue?

Reply:
because you can't trust the time that will be logged and the metrics that will be emitted.

Reply:
Ah. It is the end of validation. It makes sense to return. Sorry I had thought this gets called in the beginning of the function. 

In any case, I still think we should log the error instead of failing silently

Reply:
it's not failing, it's succeeding